erdpy/projects/templates.py

```diff
@@ -161,6 +162,34 @@ def _patch_cargo_wasm(self):
             ]
         )
 
+    def _patch_cargo_abi(self):
+        cargo_path = path.join(self.directory, "abi", TemplateRust.CARGO_TOML)
+        if not path.isfile(cargo_path):
+            return
+
+        cargo_file = CargoFile(cargo_path)
+        cargo_file.package_name = f"{self.project_name}-abi"
+        cargo_file.version = "0.0.1"
```

Comment:
I used 0.0.0 for everything in the template smart contracts, not sure what is the best way to do it though

Reply:
I'll fix this in https://github.com/ElrondNetwork/elrond-sdk/pull/142.